### ISSUE TITLE
Anusvara in ISO15919 

### ISSUE THREAD

**Issue by uberize** [virtualvinodh/aksharamukha#230]
Hi, from what I know Anusvara is represented in ISO15959 as 'ṁ' (with the dot above).  Noticed that when converting from Devanagari or ITRANS, when there is an anusvara at the end of the word (e.g. padaM in ITRANS),  Aksharamukha does convert it correctly to ṁ. 

However if you turn on the output option "Anusvara to nasal. gaṁgā → gaṅgā', it gets transliterated as 'ṃ' (with the dot below), i.e. padaṃ.

I believe this is a bug, since 'ṃ' does not really belong in ISO (it is from IAST).  You can see that if you try to convert 'padaṃ' to Devanagari, you get 'पदṃ'

To summarize, the correct transliteration of Anusvara into ISO15959 when it occurs at the end of a word should be ṁ, even when the output option is turned on.

**Comment by nsesha92** [comment]
Ref #217 seems actually ṁ is Anusvara in IAST also. I am not even a beginner, you 3 need to find the right coding.

**Comment by virtualvinodh** [comment]
Good catch! will be fixed in the next update.

**Comment by uberize** [comment]
Thank you ! Much appreciated !

SriG

**Comment by nsesha92** [comment]
Not sure how you have implemented this. Looking at the #217 discussions and links, m underdot is used in IAST in many Pali scripts in the initial days. Perhaps later it was modified to m overdot. In order to take care of both options, best way forward is to use Input option for Anusvara - m overdot or m underdot for IAST. This way both implementations can be tackled.
